libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route2/ReplayRouteInterpolator.kt

```diff
@@ -54,10 +54,11 @@ internal class ReplayRouteInterpolator {
      */
     fun createBearingProfile(replayRouteLocations: List<ReplayRouteLocation>) {
         var bearing = replayRouteLocations.first().bearing
+        val lookAhead = 2
```

Comment:
Note that we're also discussing the lookAhead for bearing. Originally was using values of 3 or 4 during development. Decided to start at 1 because there were no test cases. 

We are changing it to 2 because we're seeing this improves replay reroutes too.

Reply:
testing lookahead index with larger values is good for suburbs (where there are tight corners as shown in the u-turns video)
https://github.com/mapbox/mapbox-navigation-android/pull/2802

This value needed to be decreased after fixing curved roads. Meaning, look ahead as an index is bad for large distances (curved roads, each index is a large distance, making the look ahead very far)

lookAhead should potentially be a distance*speed, not an index. This is why I'd prefer to keep this configuration local, there are alternatives to consider

Reply:
also, this is currently a distance*speed. maybe 2 is a magic number and we want to keep it constant